pkg/nodebootstrap/userdata.go

```diff
@@ -24,6 +24,9 @@ import (
 const (
 	configDir            = "/etc/eksctl/"
 	kubeletDropInUnitDir = "/etc/systemd/system/kubelet.service.d/"
+
+	eksECRAccountStandard = "602401143452"
```

Comment:
We use this account ID for searching AMIs also, should we parametrised it everywhere?

https://github.com/weaveworks/eksctl/blob/e9fc98eef14a8bbe86f417ba10b56b9bb438964c/pkg/ami/auto_resolver.go#L53

Reply:
Thanks for the info :D. I'll update here as well. For opt-in regions, we have to use another account to host these resources. ami/ecr..